icarus/serverless.yml

```diff
@@ -155,7 +161,7 @@ resources:
           ReadCapacityUnits: 5
           WriteCapacityUnits: 10
 
-    GithubEventsTable:
+    GithubEventTable:
```

Comment:
There was a small inconsistency, as the underlying table name is plural (like others) but the env variable is singular. 
We might use plurals everywhere